be/src/common/config.h

```diff
@@ -647,6 +647,7 @@ CONF_mBool(enable_bitmap_union_disk_format_with_set, "false");
 // The number of scan threads pipeline engine.
 CONF_Int64(pipeline_scan_thread_pool_thread_num, "0");
 CONF_Int64(pipeline_hdfs_scan_thread_pool_thread_num, "48");
+CONF_Int64(pipeline_file_scan_thread_pool_thread_num, "12");
```

Comment:
Why not reuse the `scan_thread_pool`?

Reply:
In we share the `scan_thread_pool`, then blocking of external table threads (here suppose 48 external threads are blocked due to io) will also cause blocking of file read threads.